src/main/java/ch/njol/skript/conditions/CondIsInvisible.java

```diff
@@ -0,0 +1,61 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.LivingEntity;
+
+@Name("Is Invisible")
+@Description("Checks whether a living entity is invisible.")
+@Examples("target entity is invisible")
+@Since("INSERT VERSION")
+public class CondIsInvisible extends PropertyCondition<LivingEntity> {
+
+	static {
+		if (Skript.methodExists(LivingEntity.class, "isInvisible"))
+			register(CondIsInvisible.class, PropertyType.BE, "(invisible|1¦visible)", "livingentities");
```

Comment:
It's preferred to use the new parse tags (colon `:`) instead of the old parse pipes
For more info about parse tags see this [comment](https://github.com/SkriptLang/Skript/pull/4176#issuecomment-1022170530)
```suggestion
			register(CondIsInvisible.class, PropertyType.BE, "(invisible|:visible)", "livingentities");
```
This will also require a little change in `init` method which will be requested below